web/src/components/navbar/NavBarDashboardLink.test.tsx

```diff
@@ -42,12 +42,15 @@ const renderComponent = ({
   userData,
   previousBusinessId,
 }: {
-  userData?: UserData | undefined;
+  userData?: UserData | null;
```

Comment:
dust - why would userData be null?

Reply:
fixed it, was a remanent of a test I was previously trying to write